django-rgd-3d/rgd_3d/tasks/etl.py

```diff
@@ -78,3 +80,15 @@ def read_mesh_3d_file(mesh_3d):
             'vtp_data',
         ]
     )
+
+
+def read_3d_tiles_tileset_json(tiles_3d: Union[Tiles3D, int]):
+    if not isinstance(tiles_3d, Tiles3D):
+        tiles_3d = Tiles3D.objects.get(pk=tiles_3d)
+    with tiles_3d.json_file.yield_local_path() as path:
+        with open(path, 'r') as f:
+            tileset_json = json.load(f)
+    try:
+        spatial_reference = tileset_json['root']['boundingVolume']['region']
```

Comment:
@banesullivan `spatial_reference` here contains the coordinates.

Info on the `boundingVolumes` formats found in 3D tiles can be found on page 4 in this document https://github.com/CesiumGS/3d-tiles/blob/main/3d-tiles-reference-card.pdf

Reply:
I just pushed some work in progress to get this working. Its 90% there. Just need to handle transforms properly when they are given (like in the dragon example)